bedrock/products/templatetags/misc.py

```diff
@@ -19,6 +21,46 @@
 TAX_NOT_INCLUDED = ["US", "CA"]
 
 
+def _vpn_get_ga_data(bundle_relay, plan, selected_plan):
+    currency_nums = set("0123456789.,")  # just the numbers
+    id = selected_plan.get("id")
+    service = "vpn + relay" if bundle_relay else "basic"
+    period = "yearly" if plan == "12-month" else "monthly"
+    period_num = 12 if plan == "12-month" else 1
+    sale_price = selected_plan.get("price")
+    sale_price = "".join(filter(currency_nums.__contains__, sale_price))  # price as int example US$4.99 becomes: 499
+    sale_price = float(sale_price.replace(",", "."))
+    percent_savings = selected_plan.get("saving") if selected_plan.get("saving") else 0
+    percent_savings = float(percent_savings)
+    yearly_cost = sale_price * period_num
+    full_yearly_cost = yearly_cost / (1 - percent_savings / 100)
+    cost_discount = full_yearly_cost - yearly_cost
```

Comment:
I'm not sure discount is being calculated correctly here, since it should be based on the price of a standard monthly subscription in the matching currency.

Reply:
{'id' = 'price_1IXw5oKb9q6OnNsLPMkWOid7','category' = 'vpn','name' = 'basic','variant' = 'yearly','price' = '59.88','discount' = '59.88','currency' = 'EUR'}"